src/index.js

```diff
@@ -1,95 +1,132 @@
-const fs = require("node:fs");
-const path = require("node:path");
-const { Client, Collection, Events, GatewayIntentBits } = require("discord.js");
-const { token } = require("./config.json");
-
-const keepAlive = require('./serverWake.js');
-
+//for a connection with discord
+const { Client, GatewayIntentBits, messageLink, EmbedBuilder, MessageAttachment  } = require("discord.js");
+const dotenv = require("dotenv");
+const schedule = require('node-schedule');
+let dailyUpdaters = [];
+let shoutoutRule = new schedule.RecurrenceRule()
```

Comment:
don't you think dailyUpdaters array will get reset in case of bot went down ?

Reply:
but now you are using a database, so fetch the fresh data from the database itself